routers/router.js

```diff
@@ -0,0 +1,17 @@
+// currently router can only take in video id after newUrl.
+// can't show current video's id on URL yet.
+var VideoRouter = Backbone.Router.extend({
+  routes: {
+    ':id': 'fetchVidId',
+  },
+
+  fetchVidId: function(id) {
+    var allVids = appModel.get('videos');
+    var foundVidInCollection = allVids.findWhere({ _id: id });
+    if (foundVidInCollection) {
+      appModel.set('playingVideo', foundVidInCollection);
+    } else {
+      appModel.get('videos').fetchWithVidId(id);
```

Comment:
it "worked", it's because of the licensing. The link you posted is probably licensed video, since I had it set to find the closest unlicensed video. the URL does not reflect it, that's kind of what i meant by the above comment about. a bit glitchy due to licensing issue.